packages/nuxi/src/commands/devtools.ts

```diff
@@ -0,0 +1,25 @@
+import { resolve } from 'pathe'
+import { execa } from 'execa'
+import { showHelp } from '../utils/help'
+import { defineNuxtCommand } from './index'
+
+export default defineNuxtCommand({
+  meta: {
+    name: 'enable',
+    usage: 'npx nuxi devtools enable|disable [rootDir]',
+    description: 'Enable or disable features in a Nuxt project'
+  },
+  async invoke (args) {
+    const [command, _rootDir = '.'] = args._
+    const rootDir = resolve(_rootDir)
+
+    if (!['enable', 'disable'].includes(command)) {
+      console.error(`Unknown command \`${command}\`.`)
```

Comment:
Suggestion: Do we have to do this check here? If for any reason we support an additional subcommand, it requires the nuxi to be updated while same check could happen inside devtools exposed CLI. (Sorry for nitpicking :D)

Reply:
If is for usage, a fail with status code could be used if purpose is showHelp after execa. But i will do my best to make it supported directly from devtools cli. Making life much easier soon.

Reply:
Very much looking forward to seeing it! 🙏

You're right, the check here is to display usage information. When we refactor to a submodule approach it will be fully in control of devools CLI. Or we could even remove this check sooner in a patch if we had any idea of other CLI command. For now, I think it's safe.